erpnext/public/js/controllers/transaction.js

```diff
@@ -115,6 +115,12 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 			});
 		}
 
+		if(this.frm.fields_dict["items"].grid.get_field('package_tag')) {
```

Comment:
You should also check if `items` is a valid child table field

Reply:
all buying and selling cycle contain items child table